test/spdm-requester-emu/src/main.rs

```diff
@@ -369,10 +418,30 @@ fn main() {
         SOCKET_TRANSPORT_TYPE_MCTP
     };
 
-    send_receive_hello(&mut socket, transport_encap, transport_type);
+    executor::block_on(send_receive_hello(
+        socket.clone(),
+        transport_encap.clone(),
+        transport_type,
+    ));
+
+    let socket_io_transport = SocketIoTransport::new(socket.clone());
+    let socket_io_transport: Arc<Mutex<dyn SpdmDeviceIo + Send + Sync>> =
+        Arc::new(Mutex::new(socket_io_transport));
+    executor::block_on(test_spdm(
+        socket_io_transport.clone(),
+        transport_encap.clone(),
+    ));
 
-    let socket_io_transport = &mut SocketIoTransport::new(&mut socket);
-    test_spdm(socket_io_transport, transport_encap);
+    executor::block_on(send_receive_stop(socket, transport_encap, transport_type));
+}
 
-    send_receive_stop(&mut socket, transport_encap, transport_type);
+fn main() {
+    use std::thread;
+
+    thread::Builder::new()
+        .stack_size(EMU_STACK_SIZE * 1024 * 1024)
```

Comment:
This is confusing.
Why not define EMU_STACK_SIZE to be `10 * 1024 * 1024` directly?

Reply:
> This is confusing. Why not define EMU_STACK_SIZE to be `10 * 1024 * 1024` directly?

Good advice, updated.